runtime/vm/jvminit.c

```diff
@@ -6000,6 +6000,268 @@ processCompressionOptions(J9JavaVM *vm){
 	}
 }
 
+/**
+ * Parse -Xlog:gc options and add mappings, for compatibility when -XX:-LegacyXlogOption is not set.
+ * Anything besides -Xlog:gc, -Xlog:gc:stderr[:], 
+ */
+static IDATA
+parseXlogForCompatibility(J9JavaVM* vm)
```

Comment:
Please format types consistently (`J9JavaVM *vm` rather than `J9JavaVM* vm` - syntactically, the asterisk binds more tightly to the parameter name than the target type).

Reply:
Darn copied code.